### PR TITLE
Features/movie crew

### PR DESCRIPTION
**Pull request by lkkolosowski**

### CODE REVIEW

File: src/common/Images/index.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+  return (
+    <StyledLink to={toPeople()}> {/* this link will lead to a person (toPerson) in the future */}
+      <Wrapper person>
+        <Poster noPerson />
+        {poster && <Poster background={`${URL}${poster}`} />}
+      </Wrapper>
+    </StyledLink>
+  );
+};
```
Comment (Albrecht-Albi) [inline src/common/Images/index.js:39]: It's great that you created a new file :)

File: src/common/Error/index.js [hunk 2]
```diff
@@ -8,17 +8,13 @@ export const Error = () => {
...
-          <StyledLink to={toMovies()}>
-              Back to home page
-          </StyledLink>
-        </Wrapper >
+          <StyledLink to={toMovies()}>Back to home page</StyledLink>
+        </Wrapper>
       }
     />
   );
-};
```
Comment (PawelDzi) [inline src/common/Error/index.js:24]: now it's more readable

File: src/features/movieDetailsSlice.js [hunk 3]
```diff
@@ -5,12 +5,13 @@ const movieDetailsSlice = createSlice({
...
+    credits: [],
     status: "loading",
   },
   reducers: {
-    fetchMovieDetailsSuccess: (state, { payload }) => {
-      state.status = "loading";
-      state.details = payload.details;
+    fetchMovieDetailsSuccess: (state, { payload: movie }) => {
+      state.details = movie.details;
+      state.credits = movie.credits;
```
Comment (Albrecht-Albi) [inline src/features/movieDetailsSlice.js:14]: Good idea with the change

File: src/features/PeopleList/index.js [hunk 4]
```diff
@@ -1,7 +1,8 @@
-export const PeopleList = () => (
-    <>
-        People
-    </>
+import { MainWrapper } from "../../common/MainWrapper";
+import { Title } from "../../common/Title";
+
+const PeopleList = () => (
+  <MainWrapper content={<Title title="Popular People"></Title>} />
```
Comment (Albrecht-Albi) [inline src/features/PeopleList/index.js:5]: Good code style

File: src/features/MovieDetails/styled.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  &:hover {
+    transform: translateY(-6px);
+  }
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.large}) {
+    &:hover {
+      transform: translateY(-4px);
+    }
+  }
+
```
Comment (Albrecht-Albi) [inline src/features/MovieDetails/styled.js:33]: Very good style perfect with Figma

File: src/common/Images/styled.js [hunk 6]
```diff
@@ -25,9 +26,15 @@ export const Wrapper = styled.div`
...
+    person &&
+    css`
+      aspect-ratio: 3 / 4;
+    `}
+
   @media (max-width: ${({ theme }) => theme.breakpoint.large}) {
-    ${({ movieDetails }) =>
-      !movieDetails &&
+    ${({ movieDetails, person }) =>
+      !(movieDetails || person) &&
```
Comment (Albrecht-Albi) [inline src/common/Images/styled.js:37]: Good idea with the change

File: src/common/Loader/index.js [hunk 7]
```diff
@@ -1,12 +1,12 @@
...
+  <MainWrapper
+    content={
+      <Wrapper>
+        <StyledIcon />
+      </Wrapper>
+    }
+  />
 );
-
-export default Loader;
```
Comment (PawelDzi) [inline src/common/Loader/index.js:12]: a small change can make a big difference

**Review by PawelDzi — APPROVED** [review]

**Review by Albrecht-Albi — COMMENTED** [review]
Łukasz, you did a great job !!!

**Review by Albrecht-Albi — APPROVED** [review]

**Pull request merged**